xml/System.Formats.Cbor/CborWriter.xml

```diff
@@ -210,7 +222,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>
+          <see langword="true" /> if the writer has completed writing a complete root-level CBOR document, or sequence of root-level CBOR documents.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <summary>Declares whether the writer has completed writing a complete root-level CBOR document, or sequence of root-level CBOR documents.</summary>
        <value><see langword="true" /> if the writer has completed writing a complete root-level CBOR document, or sequence of root-level CBOR documents; <see langword="false" /> otherwise.</value>
```